plasma_framework/docs/design/exit_bond_and_bounty_mechanics.md

```diff
@@ -0,0 +1,190 @@
+# Exit Bond and Bounty Mechanics
+
+This document is a description of the current implementation of exit bonds and process exit bounties which are an integral part of the exit mechanism.
+
+A Process Exit Bounty is a reward which is paid out to the `processExit()` initiator in return of the transaction cost borne by the person. The motive of having a bounty is to remove an extra step of processing exits while exiting funds. This also ensures everyone pays their share of processing contrary to passing on the responsibility to someone else.
+
+The Exit Bounty is paid out as a portion (or whole) of the Exit Bond and the size of the Bond is designed to ideally cover both the cost of challenging an invalid exit or the cost of processing a valid exit 
+
+## The Bond Life Cycle
+
+### For Standard Exits
+
+The Bond is supplied by the Output owner while starting an exit and the main motive is to disincentivize invalid exits while also being able to use a portion of the bond to incentivize processing exits. The Bond is internally disintegrated into two portions. For successful exits, one of them can be returned to the output owner, and the other is a reward for processing the exit.
+The initiator/owner has to obtaint the correct size of the Exit Bond by calling `startStandardExitBondSize()`.
```

Comment:
Does it make sense to pick a terminology for the roles of the "exiting user" and "process exit initiator"?

Reply:
sure, any suggestions?